rust/Cargo.toml

```diff
@@ -53,3 +55,8 @@ quickcheck = "0.9.2"
 quickcheck_macros = "0.9.1"
 rand_os = "0.1"
 rand_chacha = "0.1"
+
+[build-dependencies]
```

Comment:
We can probably get rid of these. I was experimenting with rust build scripts but decided to just have a subcrate to run the script instead since I couldn't access the main crate from it (but I've never worked with rust build scripts so maybe it's possible). It makes more sense to only run these when we do the wasm builds/flowgen I think though anyway.